bin/zero/cli/Cargo.toml

```diff
@@ -1,12 +1,12 @@
 [package]
-name = "node-cli"
+name = "zero-cli"
 version = "3.0.0-dev"
 authors = ["ZERO <play@zero.io>"]
 description = "Generic Substrate node implementation in Rust."
 build = "build.rs"
 edition = "2021"
 license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
-default-run = "zero"
+default-run = "zero-node"
```

Comment:
not sure if we should keep the original node title `subzero`

Reply:
Should we use `subzero` instead of `zero-node`?

Reply:
would recommend to use `subzero`
we can also decide in a voting.